app/models/request/UpdatePropertyPeriodicSubmissionRequest.scala

```diff
@@ -320,7 +320,7 @@ object UpdatePropertyPeriodicSubmissionRequest {
 
     val ukOtherPropertyIncome = UkOtherPropertyIncome(
       premiumsOfLeaseGrant = propertyRentalsIncome.premiumsGrantLease.flatMap(_.premiumsGrantLease),
-      reversePremiums = propertyRentalsIncome.reversePremiumsReceived.flatMap(_.amount),
+      reversePremiums = propertyRentalsIncome.reversePremiumsReceived.flatMap(_.reversePremiums),
```

Comment:
I think this needs to stay as amount

<img width="733" alt="Screenshot 2024-12-06 at 10 36 12" src="https://github.com/user-attachments/assets/eb5f82f8-88f1-4574-8eff-ab8934429433">